pkg/apis/core/v1beta1/types_cloudprofile.go

```diff
@@ -130,6 +130,8 @@ type MachineImageVersion struct {
 	// - '< 1.26' - supports only kubelet versions less than 1.26
 	// +optional
 	KubeletVersionConstraint *string `json:"kubeletVersionConstraint,omitempty" protobuf:"bytes,4,opt,name=kubeletVersionConstraint"`
+	// InPlaceUpdateConfig contains the configuration for in-place updates for this machine image version.
+	InPlaceUpdateConfig *InPlaceUpdateConfig `json:"inPlaceUpdateConfig,omitempty" protobuf:"bytes,5,opt,name=inPlaceUpdateConfig"`
```

Comment:
WDYT about just `inPlaceUpdates` (it's clear that this is "config", like everything else)?

Reply:
Sounds good.